app/native/process/wrapped_process.cpp

```diff
@@ -1159,6 +1135,30 @@ WindowsError Process::GetExitCode(uint32_t* exit_code) {
   return WindowsError();
 }
 
+void Process::CreateMiniDump(const string& error_dump_path) {
+  HANDLE handle = CreateFileA(error_dump_path.c_str(), GENERIC_READ | GENERIC_WRITE, 0, NULL,
+    CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
+  WindowsError err = WindowsError();
+
+  if (handle == INVALID_HANDLE_VALUE) {
+    err = WindowsError("CreateMiniDump -> CreateFile", GetLastError());
```

Comment:
This should be reorganized so that the code won't call MiniDumpWriteDump if an error occurs here

Reply:
Well, now we're losing the error without logging it.
It probably won't ever be an important error path, so if you have had enough of revising this code then feel free to just merge it =) But this possibly should have an inner function which returns the error that is then checked. (Or a try-catch, but maybe there was a reason why this code doesn't use exceptions)

Reply:
Good point.

I fixed it now, although in a bit uglier way than you suggested. I think it's fine for now as it's unlikely this will have to change much in future, if at all?

Reply:
Yup that's perfectly fine now